pages/apis/rest_api.md

```diff
@@ -49,12 +49,20 @@ You can authenticate with the Buildkite API using access tokens.
 
 API access tokens allow to call the API without using your username and password. They can be created on your <a href="<%= url_helpers.user_access_tokens_url %>" rel="nofollow">API access tokens</a> page, limited to individual organizations and permissions, and revoked at any time from the web interface [or the REST API](/docs/apis/rest-api/access-token#revoke-the-current-token).
 
-To authenticate using a token, set the <code>Authorization</code> HTTP header to the word <code>Bearer</code>, followed by a space, followed by the access token. For example:
+Authentication can be done with both HTTP headers and as a query parameter in a HTTP request.
```

Comment:
Style: Let's use the active voice here. I think it would also be good to highlight that you'll do one or the other, not both.

```suggestion
You can include the access token in a request using either an HTTP header or a query parameter.
```